examples/async/index.js

```diff
@@ -1,15 +1,34 @@
 import 'babel-polyfill'
 import React from 'react'
-import { render } from 'react-dom'
+import ReactDOM from 'react-dom'
 import { Provider } from 'react-redux'
-import App from './containers/App'
 import configureStore from './store/configureStore'
+import App from './containers/App'
 
 const store = configureStore()
+const rootEl = document.getElementById('root')
+
+function render(rootReactEl) {
+  ReactDOM.render(
+    <Provider store={store}>
+      {rootReactEl}
+    </Provider>,
+    rootEl
+  )
+}
```

Comment:
Ah, good call. My goal was to make sure RedBox doesn't get into the production build but I can put some more conditionals to ensure that.